mobidam-sst-management-frontend/apigateway/src/main/resources/application.yml

```diff
@@ -56,3 +76,9 @@ info.application.version: @project.version@
 info.appswitcher.url: https://appswitcher.muenchen.de
 
 config.map5xxto400: false
+
+app:
+  gateway:
+    routing:
+      backend:
+        uri: http://localhost:39146/
```

Comment:
rauslöschen oder auslagern in eine local/test-YML. @martind260 hast du eine Idee?